pyproject.toml

```
@@ -50,7 +50,7 @@ awkward = [
 ]
 dev = [
   "awkward>=1.2",
-  'numba>=0.50; python_version < "3.11"',
+  'numba>=0.57; python_version > "3.7"',
```

Numba 0.57 isn’t available for Python 3.7.

---

Oh wait, saying `python_version > "3.7"`  will install it on Python 3.7.x where x > 0?

---

Ah, no, `python_version` is two digits, so this will actually work. But please use:

```suggestion
  'numba>=0.57; python_version >= "3.8"',
```

As it’s clearer, imo. :)

---

Ah, thanks for the quick explanation!